lib/core/macrocache.nim

```diff
@@ -143,6 +143,19 @@ iterator items*(s: CacheSeq): NimNode =
 
   for i in 0 ..< len(s): yield s[i]
 
+proc contains*(s: CacheSeq, target: NimNode): bool {.inline.} =
```

Comment:
Problematic because `NimNode` has reference semantics. Usually one would create a tree of identical shape though and then using `contains` cannot work.